jupyter-docker/Dockerfile

```diff
@@ -147,6 +149,8 @@ RUN apt-get install -yq --no-install-recommends python3 \
  && pip3 install seaborn \
  && pip3 install python-lzo \
  && pip3 install google-cloud \
+ # pydot is a dependency of firecloud; peg the version because pydot >=1.2.2 requires a version of pyparsing which conflicts with the distribution
+ && pip install 'pydot<1.2.2' \ 
```

Comment:
Should this one be pip3?

Reply:
yes but I've ditched in this in favor of a better fix